### PR TITLE
Support disabling ordering doc tags in command line

### PR DESCRIPTION
**Pull request by omarismail94**
Tested: Added unit test

Also compiled the JAR and tested it running:
```
 java -jar ./ktfmt-0.50-SNAPSHOT-jar-with-dependencies.jar --kotlinlang-style --do-not-order-doc-tags compose/runtime/runtime-saveable/src/commonMain/kotlin/androidx/compose/runtime/saveable/RememberSaveable.kt
```
and it did not re-order the kdcoc tags in the kotlin file, as expected

Fixed: Issue #406

### CODE REVIEW

**Comment by omarismail94** [comment]
Signed CLA

**Review by hick209 — APPROVED** [review]

File: core/src/main/java/com/facebook/ktfmt/kdoc/KDocFormattingOptions.kt [hunk 1]
```diff
@@ -41,7 +41,12 @@ class KDocFormattingOptions(
     /**
      * Limit comment to be at most [maxCommentWidth] characters even if more would fit on the line.
      */
-    var maxCommentWidth: Int = min(maxLineWidth, 72)
+    var maxCommentWidth: Int = min(maxLineWidth, 72),
+
+    /**
+     * If true, moves any kdoc tags to the end of the comment and `@return` tags after `@param` tags.
+     */
+    var orderDocTags: Boolean = true
```
[the remark this answers is not in the document]
Comment (omarismail94) [reply core/src/main/java/com/facebook/ktfmt/kdoc/KDocFormattingOptions.kt]: Done, thanks!

**Review by hick209 — APPROVED** [review]

**Review by hick209 — APPROVED** [review]

**Review by hick209 — CHANGES_REQUESTED** [review]
I'm actually not sure we want this option to be exposed as a CLI param.
I'm discussing it internally at first, but wanted to give you a heads up

**Comment by hick209** [comment]
I understand why the feature is needed, but not why you need direct CLI exposure yet.
A workaround for it would be to build a simple Java program and runs ktfmt for you, where you set it up appropriately, if you don't want to touch the Google style

**Comment by omarismail94** [comment]
> I understand why the feature is needed, but not why you need direct CLI exposure yet. A workaround for it would be to build a simple Java program and runs ktfmt for you, where you set it up appropriately, if you don't want to touch the Google style

Ah OK, so I can keep the feature exposed in FormattingOptions, but not via CLI? 

I wanted to expose in CLI as that way, if I were to package the JAR as in intellij plugin, I can give it the flag to not re-order the tags. This would be a nice to have.

**Comment by omarismail94** [comment]
I changed it so `orderDocTags` is only in FormattingOptions and not avaialble via CLI

**Comment by omarismail94** [comment]
Im going to close this in preference for PR #475

**Pull request closed without merging**